navigation/ca_cartographer/launch/pure_localization.launch

```diff
@@ -0,0 +1,22 @@
+<launch>
+  <param name="/use_sim_time" value="true" />
+
+  <arg name="load_state_filename" default="$(find ca_cartographer)/house_map/house_map.pbstream" />
```

Comment:
I recommend you to change it like this:

```xml
<arg name="pbstream" default="$(optenv PBSTREAM house_map)"/>
<arg name="load_state_filename" value="$(find ca_cartographer)/house_map/$(arg pbstream).pbstream" />
```

Now, there is no need to set `load_state_filename` since that argument contains the key `value` instead of `default`.